packages/synapse-react-client/src/components/TableColumnSchemaEditor/ColumnModelFormFields/DefaultValueField.test.tsx

```diff
@@ -168,20 +187,27 @@ describe('DefaultValueField', () => {
 
     const multiValueDialog = await screen.findByRole('dialog')
 
-    const itemTextFields = await within(multiValueDialog).findAllByRole(
-      'textbox',
+    const datePickerFields = await within(multiValueDialog).findAllByTestId(
+      MOCK_DATETIME_PICKER_TEST_ID,
     )
+    expect(datePickerFields).toHaveLength(2)
 
-    expect(itemTextFields).toHaveLength(2)
-    expect(itemTextFields[0].getAttribute('value')).toMatch(/2023/)
-    expect(itemTextFields[1].getAttribute('value')).toMatch(/2024/)
+    expect(mockDateTimePicker).toHaveBeenCalledWith(
+      expect.objectContaining({
+        value: dayjs(DATE_ONE),
+      }),
+      expect.anything(),
+    )
+    expect(mockDateTimePicker).toHaveBeenCalledWith(
+      expect.objectContaining({
+        value: dayjs(DATE_TWO),
+      }),
+      expect.anything(),
+    )
 
     await userEvent.click(screen.getByRole('button', { name: 'OK' }))
 
-    // Saving will convert these to ISO strings
-    expect(onChange).toHaveBeenCalledWith([
-      dayjs(1678852800000).toISOString(),
-      dayjs(1710475200000).toISOString(),
-    ])
+    // Saving should invoke onChange, which should return the original type, millisecond timestamps
+    expect(onChange).toHaveBeenCalledWith([DATE_ONE, DATE_TWO])
```

Comment:
Slight change to the DATE_LIST case. The data passed to `onChange` now matches the original format--integer timestamps